### ISSUE TITLE
Preform holder

### ISSUE THREAD

**Issue by joycopl** [copl-labomc/fabconcept#23]
New Preform holder should be capable to hold preform easily with variable preform diameter.

**Comment by cniallen** [comment]
Good point, I was forgetting that one 👍 We should not have to design that whole holder from scratch, mechanical systems that hold drill bits (mandrin de Jacob in French, I don’t know the English name unfortunately) are in principle commercially available in all sorts of sizes. Again the team at the mechanical workshop can give good advices here, feel free to go see them anytime 😃

**Comment by alexmomo1212** [comment]
Speak to the mechanical workshop to see if the have ideas/products.
